test/infrustructure/receipt/CreateReceiptFromDTO.spec.ts

```diff
@@ -0,0 +1,149 @@
+import {Account} from '../../../src/model/account/Account';
+import { Address } from '../../../src/model/account/Address';
+import { NetworkType } from '../../../src/model/blockchain/NetworkType';
+import { MosaicId } from '../../../src/model/mosaic/MosaicId';
+import { AddressAlias } from '../../../src/model/namespace/AddressAlias';
+import { MosaicAlias } from '../../../src/model/namespace/MosaicAlias';
+import { ReceiptType } from '../../../src/model/receipt/ReceiptType';
+
+describe('Receipt - CreateStatementFromDTO', () => {
+    let account: Account;
+    let account2: Account;
+    let statementDto;
+    const netWorkType = NetworkType.MIJIN_TEST;
+
+    before(() => {
+        account = Account.createFromPrivateKey('D242FB34C2C4DD36E995B9C865F93940065E326661BA5A4A247331D211FE3A3D', NetworkType.MIJIN_TEST);
+        account2 = Account.createFromPrivateKey('E5DCCEBDB01A8B03A7DB7BA5888E2E33FD4617B5F6FED48C4C09C0780F422713', NetworkType.MIJIN_TEST);
+        statementDto = {transactionStatements:  [
+            {
+                height: [52, 0],
+                source: {
+                  primaryId: 0,
+                  secondaryId: 0,
+                },
+                receipts: [
+                    {
+                        version: 1,
+                        type: 8515,
+                        account: account.publicKey,
+                        mosaicId: [3646934825, 3576016193],
+                        amount: [1000, 0],
+                    },
+                ],
+            },
+        ],
+        addressResolutionStatements: [
+            {
+                height: [1488, 0],
+                unresolved: '9103B60AAF2762688300000000000000000000000000000000',
+                resolutionEntries: [
+                  {
+                    source: {
+                      primaryId: 4,
+                      secondaryId: 0,
+                    },
+                    resolved: '917E7E29A01014C2F300000000000000000000000000000000',
+                  },
+                ],
+            },
+            {
+                height: [1488, 0],
+                unresolved: '917E7E29A01014C2F300000000000000000000000000000000',
+                resolutionEntries: [
+                  {
+                    source: {
+                      primaryId: 2,
+                      secondaryId: 0,
+                    },
+                    resolved: '9103B60AAF2762688300000000000000000000000000000000',
+                  },
+                ],
+            },
+        ],
+        mosaicResolutionStatements: [
+            {
+                height: [
+                  1506,
+                  0,
+                ],
+                unresolved: [4014740460, 2448037180],
+                resolutionEntries: [
+                  {
+                    source: {
+                      primaryId: 1,
+                      secondaryId: 0,
+                    },
+                    resolved: '08a12f89ee5a49f8',
```

Comment:
how come the format here is different from an address resolution statement ?

Reply:
that schema has been changed to UInt64 recently. It's now fixed